sdk/communication/AzureCommunication/Source/CommunicationCloudEnvironment.swift

```diff
@@ -0,0 +1,66 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import Foundation
+
+#if canImport(AzureCore)
+    import AzureCore
+#endif
+import Foundation
+
+@objcMembers public class  CommunicationCloudEnvironment {
```

Comment:
I assume this is @objcMembers cause we expect calling to use this?

Reply:
`CommunicationCloudEnvironment ` is now a property of MicrosoftTeamsUserIdentifier. And MicrosoftTeamsUserIdentifier, is used by calling. 
so that is why I add @objcMembers here too ,

![image](https://user-images.githubusercontent.com/8701731/105885053-7a491680-5fbd-11eb-87fd-0416ceb58099.png)

Reply:
that's too bad I think this would be great to use an enum instead of class